spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -94,69 +94,69 @@ Use of implementations beyond Jakarta Annotations 1.3 in MicroProfile is allowed
  - https://github.com/eclipse-ee4j/common-annotations-api
 
 [[mp-config]]
-=== Eclipse MicroProfile Config 2.0-M1
+=== MicroProfile Config 2.0-RC1
 
-Eclipse MicroProfile Config provides applications and microservices means to obtain configuration properties through several environment-aware sources both internal and external to the application and made available through dependency injection or lookup.
+MicroProfile Config provides applications and microservices means to obtain configuration properties through several environment-aware sources both internal and external to the application and made available through dependency injection or lookup.
 
  - http://microprofile.io/project/eclipse/microprofile-config
- - https://github.com/eclipse/microprofile-config/releases/tag/2.0-M1
+ - https://github.com/eclipse/microprofile-config/releases/tag/2.0-RC1
 
 [[mp-fault-tolerance]]
-=== Eclipse MicroProfile Fault Tolerance 3.0-RC1
+=== MicroProfile Fault Tolerance 3.0-RC1
 
-Eclipse MicroProfile Fault Tolerance provides the ability to separate execution logic from business logic.
+MicroProfile Fault Tolerance provides the ability to separate execution logic from business logic.
 Key aspects of the API include TimeOut, RetryPolicy, Fallback, Bulkhead, and Circuit Breaker processing.
 Fault Tolerance 3.0 is now dependent on Jakarta EE 8 technologies.
 
  - http://microprofile.io/project/eclipse/microprofile-fault-tolerance
  - https://github.com/eclipse/microprofile-fault-tolerance/releases/tag/3.0-RC1
 
 [[mp-health-check]]
-=== Eclipse MicroProfile Health 3.0-RC3
+=== MicroProfile Health 3.0-RC3
```

Comment:
we have now RC5 available -> https://search.maven.org/artifact/org.eclipse.microprofile.health/microprofile-health-api/3.0-RC5/jar

Reply:
Ah yes I see, I have updated the RC versions of all specifications for good measure.